src/Components/ComponentAlreadyAssignedException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Components
+{
+	public class ComponentAlreadyAssignedException : Exception
+	{
+		private Type _entityType;
```

Comment:
Yes, there is zero use for them being stored like this as they can't be accessed! Should be public Type {get; private set;}